src/sentry/utils/raven.py

```diff
@@ -34,16 +35,26 @@ def is_enabled(self):
             return False
         return settings.SENTRY_PROJECT is not None
 
-    def capture(self, *args, **kwargs):
+    def send(self, **kwargs):
+        # TODO(dcramer): this should respect rate limits/etc and use the normal
+        # pipeline
+
+        # Report the issue to an upstream Sentry if active
+        # NOTE: we don't want to check self.is_enabled() like normal, since
+        # is_enabled behavior is overridden in this class. We explicitly
+        # want to check if the remote is active.
+        if self.remote.is_active():
+            from sentry import options
+            # Append some extra tags that are useful for remote reporting
+            super_kwargs = copy.deepcopy(kwargs)
```

Comment:
I don't like making this copy, since this is a relatively large object, but I wanted to make sure that any mutation within the `super()` call didn't affect the internal reporting, so this was most safe.